web/src/desktop_notifications.ts

```diff
@@ -91,9 +91,10 @@ export function permission_state(): string {
     return NotificationAPI.permission;
 }
 
-export function close_notification(message: Message): void {
+export function close_notification(message: Message | number): void {
+    const id = typeof message === "number" ? message : message.id;
     for (const [key, notice_mem_entry] of notice_memory) {
-        if (notice_mem_entry.message_id === message.id) {
+        if (notice_mem_entry.message_id === id) {
```

Comment:
It'd be better to make this function just accept an integer ID unconditionally, and have the one existing caller use the new protocol. 

@evykassirer would you have time to take over finishing this issue?

Reply:
Yeah I can

Reply:
Hey @evykassirer, if you don't mind can I complete this issue ?

Reply:
If you complete this issue, you'll need to reply to Tim's comment above "OK, and how have you tested this?". Did you reproduce the issue locally first? Did you see that it went away after this code change?